manifests/s/Syncthing/Syncthing/1.23.3/Syncthing.Syncthing.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Syncthing.Syncthing
+PackageVersion: 1.23.3
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: syncthing-windows-amd64-v1.23.3\syncthing.exe
```

Comment:
Since this RelativeFilePath contains part of the package name, it's only valid for the x64 package. Installing the x86 version in a Windows 11 VM gives this error:
> 2023-04-04 08:51:19.364 [CLI ] Unable to locate nested installer at: C:\Users\User\AppData\Local\Temp\WinGet\Syncthing.Syncthing.1.23.3\extracted\syncthing-windows-amd64-v1.23.3\syncthing.exe

This is because it's under `syncthing-windows-386-v1.23.3\syncthing.exe` - with `386` replacing `amd64`. 

I'm not sure of the best way to fix this. I believe most portable packages have just one installer.